config/optional/access_unpublished.settings.yml

```diff
@@ -0,0 +1,5 @@
+hash_key: review
+duration: 172800
+cleanup_expired_tokens: false
+cleanup_expired_tokens_period: '-2 weeks'
+modify_http_headers: {  }
```

Comment:
Move this whole file to `az_core/config/optional`

Reply:
Doesn't look like it works right in this location. Need to move it back to `config/optional`.